File: bakjoon/python/implementation/1316.py
n=int(input())
ans=0
for _ in range(n):
    word=input()
    is_true=0
    for i in range(len(word)-1):
        if word[i]!=word[i+1]:
            cur=word[i]
            for j in range(i+1,len(word)):
                if cur==word[j]:
                    is_true=1
                    break
            if is_true:
                break
    if not is_true:
        ans+=1
print(ans)

File: 2022/bakjoon/kotlin/implementation/1316.kt
package com.example.bakjoon

fun main() = with(System.`in`.bufferedReader()) {
    val n = readln().toInt()
    var ans = 0
    repeat(n) {
        val word = readLine().toCharArray()
        var is_true = false
        for (i in 0..word.size-2){
            if (word[i]!=word[i+1]){
                val cur = word[i]
                for (j in i+1..word.size-1){
                    if (cur==word[j]){
                        is_true=true
                        break
                    }
                }
                if (is_true){
                    break
                }
            }
        }
        if (!is_true){
            ans++
        }

    }
    print(ans)
}